datamodel.ttl

```diff
@@ -11,34 +11,48 @@
 <http://emmo.info/datamodel> rdf:type owl:Ontology ;
                               owl:versionIRI <http://emmo.info/datamodel/0.0.2> ;
                               owl:imports <http://emmo.info/datamodel/0.0.2/collection> ,
-                                          <http://emmo.info/emmo/1.0.0-beta4/perspectives/data> ,
+                                          <http://emmo.info/emmo/1.0.0-beta4/disciplines/math> ,
                                           <http://emmo.info/emmo/1.0.0-beta4/multiperspective/persholistic> ,
                                           <http://emmo.info/emmo/1.0.0-beta4/multiperspective/symbolic> ,
-                                          <http://emmo.info/emmo/1.0.0-beta4/disciplines/math> ;
+                                          <http://emmo.info/emmo/1.0.0-beta4/perspectives/data> ;
                               dcterms:abstract "This module connects the SOFT7 data model and the metamodel described in entity.ttl and metamodel.ttl, respectively, to EMMO."@en ;
                               dcterms:creator "Jesper Friis, SINTEF"@en ,
                                               "Thomas Hagelien, SINTEF"@en ;
                               dcterms:license "https://creativecommons.org/licenses/by/4.0/legalcode" ;
                               dcterms:title "Datamodel ontology"@en .
 
+#################################################################
+#    Annotation properties
+#################################################################
+
+###  http://purl.org/dc/termsabstract
+dcterms:abstract rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/termscreator
+dcterms:creator rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/termslicense
+dcterms:license rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/termstitle
+dcterms:title rdf:type owl:AnnotationProperty .
 
 
 #################################################################
 #    Object Properties
 #################################################################
 
 ###  http://emmo.info/datamodel#aggregation
-:aggregation rdfs:subPropertyOf emmo:EMMO_6703954e_34c4_4a15_a9e7_f313760ae1a8 .
+:aggregation rdfs:subPropertyOf emmo:EMMO_8785be5a_2493_4b12_8f39_31907ab11748 .
 
 
 ###  http://emmo.info/datamodel#composition
 :composition rdfs:subPropertyOf emmo:EMMO_d893d373_b579_4867_841e_1c2b31a8d2c6 .
 
 
-###  http://emmo.info/datamodel#dataModelRelation
```

Comment:
Why is dataModelRelation removed? Whjat was it used for before?

Reply:
> Why is dataModelRelation removed? Whjat was it used for before?

It was removed because it wasn't used before.